### DISCUSSION TITLE
Domain expired, can't manually renew

### DISCUSSION THREAD

**Discussion by scott-riley** [vercel/community#4411] [Help] [answered]
### Summary

I had old bank details configured and forgot to update them until the day my domain was set to expire. Assuming the auto-renew didn't kick in resulting in the domain being allowed to expire.

Anything I can do or did I leave things too late?

Thanks!

### Steps to Reproduce

Let a domain expire, panic 😅

**Comment by lucas-0liveira** [comment]
Certainly, I understand the situation. It seems that I had old bank details configured and ended up forgetting to update them until the day my domain was set to expire. Assuming the auto-renewal didn't kick in, resulting in the domain being allowed to expire. It was my oversight that led to this, unfortunately. I will take steps to ensure this doesn't happen again in the future.

**Comment by mcsdevv** [comment, marked as answer]
Hey folks, we're planning some improvements to make this manageable end to end for customers because I totally get it's a less than ideal situation. For now, open up a case at vercel.com/help under 'Billing' and we'll help out as soon as we can!

**Reply by lucas-0liveira** [reply]
The suggestion to open a case at vercel.com/help under the 'Billing' section is very helpful and provides a clear path for customers to get assistance. Great initiative!

**Comment by kitavidavis** [comment]
Hello @mcsdevv , my vercel domain just expired. How can I renew it on the vercel platform? I am using the freemium version

**Reply by mcsdevv** [reply]
Hey @kitavidavis, are you able to follow the guidance outlined in the selected answer and open up a case at vercel.com/help under 'Billing'?

**Reply by kitavidavis** [reply]
Yes, I have submitted a billing case but I have not yet received any feedback. Is there a way I can accelerate this?

**Reply by mcsdevv** [reply]
Yep, cases found, merged, and you'll receive a reply shortly.

**Reply by kitavidavis** [reply]
Alright. Waiting
